test/core-test.js

```diff
@@ -82,6 +81,34 @@ describe("EncodeDAOCore", function () {
 
   })
 
+    it("Should complete accepted Issue", async function () {
+    var strBytes = new Uint8Array("fix roof");
+    await encodeDAOCore.connect(addr1).proposeIssue(strBytes, 50, "We need to fix the roof - it's raining on my head!")
+    var issueId = 0;
+    await encodeDAOCore.connect(addr1)
+      .voteIssue(issueId, true)
+    await encodeDAOCore.connect(addr2)
+      .voteIssue(issueId, true)
```

Comment:
It appears both are voting the same, is this just to check if it can be voted on by more than 1 person?